Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -204,3 +204,17 @@ the size of the cluster.
     ceph orch set backend ""
     ceph mgr module disable rook
     ```
+
+## External Clusters
+
+User Applications can make use of the NFS client running on the external ceph standalone cluster.
+
+Procedure:
+
+Export the nfs client to a particular cephfs filesystem:
+
+```yaml
+ceph nfs export create cephfs <nfs-client-name> /test <filesystem-name>
+```
```

Comment:
To be honest, I don't think Rook should document this. Users should defer to Ceph docs for instructions on how to create exports.

Instead, Rook docs should explain how to use a Ceph export's configurations (maybe it's already existing, maybe users create it just for Rook, it doesn't matter) in the PV/PVC. 

I think this command should more rightly be a `ceph nfs export get <service> <export-name>` command, and then we can show how to use the example output of that command to configure the PV/PVCs.